### ISSUE TITLE
Companion APP with URL and hash

### ISSUE THREAD

**Issue by joaopedros2** [Clooos/Bubble-Card#270]
Hello, I'm trying to migrate things to have just one 'view' in the homepage lovelace.

I attempted to use the 'hash' directly in the URL of my camera notifications, but unfortunately, it doesn't work; it ignores the 'hash.'

Is there another way to achieve this?

```
service: notify.mobile_app_phone
data:
  title: "Motion Detected"
  message: "Someone."
  data:
    clickAction: '/lovelace/homepage#cameras'
```

**Comment by joaopedros2** [comment]
I tested it with version v1.6.0-beta.4, and changed yaml to "bubble-pop-up.js?v1.6.0-beta.4" but doesn’t work.

**Comment by pressurejets** [comment]
Cant you just make a new view page 'mobileappdoorcam' and hide the view ?? Then you will still only have one view page and hidden pages. anyways that the way i did it..

**Comment by joaopedros2** [comment]
> mobileappdoorcam

I didn't quite understand what you did to make it work.

**Comment by joaopedros2** [comment]
> Cant you just make a new view page 'mobileappdoorcam' and hide the view ?? Then you will still only have one view page and hidden pages. anyways that the way i did it..

**Comment by joaopedros2** [comment]
Yes

**Comment by joaopedros2** [comment]
> Hi! I've tried again, and it's working but only on a pop-up that don't have any trigger entity.
> 
> Edit: But I've just fixed this issue, just wait for the v2.2.2, and sorry for the very late answer 😅
> 
> Edit 2: The v2.2.2 is here!
> 
> https://github.com/Clooos/Bubble-Card/releases/tag/v2.2.2

it works, thanks man!
